tests/lib/Content/Form/Provider/GroupedContentFormFieldsProviderTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\ContentForms\Content\Form\Provider;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\TextLine\Value;
+use eZ\Publish\Core\Helper\FieldsGroups\FieldsGroupsList;
+use eZ\Publish\Core\Repository\Values\ContentType\FieldDefinition;
+use EzSystems\EzPlatformContentForms\Data\Content\FieldData;
+use Ibexa\ContentForms\Content\Form\Provider\GroupedContentFormFieldsProvider;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormInterface;
+
+class GroupedContentFormFieldsProviderTest extends TestCase
+{
+    public function testGetGroupedFields()
+    {
+        $fieldsGroupsListMock = $this->getMockBuilder(FieldsGroupsList::class)->getMock();
```

Comment:
If you're not actually configuring the `MockBuilder`, then it's not necessary to include that intermediate step.
```suggestion
        $fieldsGroupsListMock = $this->createMock(FieldsGroupsList::class);
```